### ISSUE TITLE
Unresolved reference make_ASC_raster in times_prt

### ISSUE THREAD

**Issue by pesekon2** [storm-fsv-cvut/smoderp2d#211]
Variable `make_ASC_raster` used [here](https://github.com/storm-fsv-cvut/smoderp2d/blob/master/smoderp2d/tools/times_prt.py#L50) is not defined anywhere.

**Comment by pesekon2** [comment]
The same non-existing function is apparently called also in tools.py, [here](https://github.com/storm-fsv-cvut/smoderp2d/blob/master/smoderp2d/tools/tools.py#L23).

**Comment by pesekon2** [comment]
@jerabekjak: Please leave a comment what the hell it is.

**Comment by jerabekjak** [comment]
The times_prt function was supposed to print rasters at some specific times. not only at the end of the computation. It worked long ago. it is not implemented now. However, generally, we would like to have such functionality.

Maybe, at this point, it is easier to write something new from scratch instead of updating the make_asc_raster and times_prt...

What do you think? @pesekon2

**Comment by pesekon2** [comment]
@jerabekjak: I agree that it would be a valuable functionality. Also agree that it would probably be easier to write it from scratch as the data structure has changed since.
